bin/yaml/android-java.yaml

```diff
@@ -10,3 +10,20 @@ compilers:
     url: https://dl.google.com/android/maven2/com/android/tools/r8/{name}/r8-{name}.jar
     targets:
       - 8.1.56
+
+  nightly:
+    if: nightly
+    type: script
+    dir: "{name}"
+    fetch:
```

Comment:
is this fetch meant to be empty?

Reply:
Ahh, this line was meant to be deleted... I had added it while testing out the change to script.py, but had forgotten to clean it up again. Sorry about that! I see that you fixed it in a later change—thank you!